extensions/wikia/PortableInfobox/services/Parser/Nodes/NodeImage.php

```diff
@@ -76,6 +97,40 @@ private function getImageAsTitleObject( $imageName ) {
 		return $title;
 	}
 
+	private function getGalleryItems( $imageData ) {
+		preg_match( '/.(UNIQ.*QINU)./U', $imageData, $galleryMarkers );
+		$galleryImages = array();
+
+		foreach ( $galleryMarkers as $marker ) {
+			$galleryWikitext = PortableInfoboxDataBag::getInstance()->getGallery( $marker );
+			$galleryLines = preg_split('/\r\n|\n|\r/', $galleryWikitext);
+
+			foreach ( $galleryLines as $galleryImage) {
```

Comment:
@dianafa I sent you couple of messages on slack regarding this change (based on conversations we had before) - but high level summary is: for now I'm sticking to regexes instead of creating another yet parser for galleries. Ultimately it would be good to have one solid parser for galleries and let's talk about it during some sort of retrospective - after we release this project for our beta communities.